maven-plugin-report-plugin/pom.xml

```diff
@@ -101,13 +100,30 @@
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-utils</artifactId>
     </dependency>
+    <!-- Plexus -->
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-i18n</artifactId>
+      <version>1.0-beta-10</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.codehaus.plexus</groupId>
+          <artifactId>plexus-component-api</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
 
     <!-- tests -->
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-api</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Why you add this dependencies?

Reply:
Because otherwise the test fails for me. At least, when added all tests work.

Reply:
Remove and see whether it works for you.

Reply:
I am happy to reduce as many dependencies as possible.

Reply:
Are we good here or should I pursue another reduce? Before Doxia 2.0.0 the effort isn't really worth it.

Reply:
looks as works after remove for me - I can try clean it later.